includes/class-evf-form-task.php

```diff
@@ -292,7 +292,7 @@ public function do_task( $entry ) {
 		}
 
 		$message = isset( $this->form_data['settings']['successful_form_submission_message'] ) ? $this->form_data['settings']['successful_form_submission_message'] : __( 'Thanks for contacting us! We will be in touch with you shortly.', 'everest-forms' );
-		if ( 1 == $ajax_form_submission ) {
+		if ( 1 === $ajax_form_submission ) {
```

Comment:
@roshansameer Please look if the AJAX Form submission setting is effected or not?